crypto/evp/evp_lib.c

```diff
@@ -19,31 +19,39 @@
 #include "evp_locl.h"
 
 #if !defined(FIPS_MODE)
+static int asn1_flags(const EVP_CIPHER *cipher)
+{
+    /*
+     * If this is a provided implementation and EVP_CIPH_FLAG_CUSTOM_ASN1 is
+     * set, the implementation has its own handling of AlgorithmIdentifier
+     * parameters.
+     * If EVP_CIPH_FLAG_DEFAULT_ASN1 is set, the implementation is asking
+     * libcrypto to handle the ASN.1 params with default libcrypto methods.
+     * Otherwise.  This is a legacy flag.
```

Comment:
We seem to have sentence fragment here "Otherwise".

Reply:
Oops, eyah